packages/compose-tunnel-agent/src/docker.ts

```diff
@@ -32,6 +33,7 @@ const client = ({
   ).map(x => ({
     project: x.Labels['com.docker.compose.project'],
     name: x.Labels['com.docker.compose.service'],
+    access: (x.Labels['preevy.access'] || 'public') as ('private' | 'public'),
```

Comment:
How would "saas-private" access work? isn't this another value of `preevy.access`?

Reply:
depends on what saas do u refer to.. but we can always add authentication type.. wether it's basic or other